drivers/python/rethinkdb/net.py

```diff
@@ -26,6 +27,11 @@ def import_module(name, package=None):
 from .ast import recursively_convert_pseudotypes
 
 try:
+    from ssl import match_hostname, CertificateError
+except ImportError:
+    from backports.ssl_match_hostname import match_hostname, CertificateError
```

Comment:
As mentioned, I think we should make this import optional in case the dependency isn't installed.